packages/manager/apps/telecom/src/app/telecom/pack/slots/hostedEmail/detail/index.js

```diff
@@ -0,0 +1,23 @@
+import angular from 'angular';
+
+import ngTranslateAsyncLoader from '@ovh-ux/ng-translate-async-loader';
+import uiRouter from '@uirouter/angularjs';
+import angularTranslate from 'angular-translate';
+import 'ovh-ui-angular';
+
+import component from './detail.component';
+import routing from './detail.routing';
+
+const moduleName = 'ovhTelecomPackHostedEmailDetail';
+
+angular
+  .module(moduleName, [
+    ngTranslateAsyncLoader,
+    uiRouter,
+    angularTranslate,
+    'oui',
+  ])
+  .component('detail', component)
```

Comment:
⚠️ Avoid using generic component name.

Here it could reflect the place where this component is used for instance:

```suggestion
  .component('packSlotHostedEmailDetail', component)
```

What's your opinion?